registration/templates/base.html

```diff
@@ -37,6 +37,7 @@
     ></script>
   </head>
   <body>
+    {% include 'navbar.html' %}
```

Comment:
I do not see any tests for the navbar. Was this part of the ticket?

Reply:
This wasn't part of a ticket. I just added it for better navigation. I added a test with [2b694c2](https://github.com/gocept/ok_tools/pull/16/commits/2b694c274f570b3b775ecec38c22e83ee4e5e07f).